spyder/utils/programs.py

```diff
@@ -288,7 +287,10 @@ def run_python_script_in_terminal(fname, wdir, args, interact,
                                    "an external terminal"),
                                  QMessageBox.Ok)
     elif os.name == 'posix':
-        programs = [{'cmd': 'gnome-terminal',
+        programs = [{'cmd': 'x-terminal-emulator',
+                     'wdir-option': '--working-directory',
+                     'execute-option': '-x'},

```

Comment:
Actually no, I observed that external scripts are opened on a different terminal instead on the system default one. On my case I use Tilix, but it opens gnome-terminal, so I thought it would be better to open the custom one. However I can remove this one